core/Filesystem.php

```diff
@@ -30,6 +31,7 @@ public static function deleteAllCacheOnUpdate($pluginName = false)
         TrackerCache::deleteTrackerCache();
         PiwikCache::flushAll();
         self::clearPhpCaches();
+        ServerFilesGenerator::createFilesForSecurity();
```

Comment:
done in https://github.com/piwik/piwik/commit/bd095961c13249dcafbb283e5da9e028e31bd428